operator/pkg/securedcluster/reconciler/reconciler.go

```diff
@@ -29,15 +29,15 @@ func RegisterNewReconciler(mgr ctrl.Manager, selector string) error {
 			// Only appearance and disappearance of a Central resource can influence whether
 			// a local scanner should be deployed by the SecuredCluster controller.
 			utils.CreateAndDeleteOnlyPredicate{}),
-		pkgReconciler.WithPreExtension(extensions.CheckClusterNameExtension(nil)),
-		pkgReconciler.WithPreExtension(proxy.ReconcileProxySecretExtension(mgr.GetClient(), proxyEnv)),
+		pkgReconciler.WithPreExtension(extensions.CheckClusterNameExtension(nil, nil)),
```

Comment:
Why are we passing `nil` here ?

Reply:
Because this extension does not send any requests to K8s. It is still using wrapExtension internally so I stuck to the passing nil value.

I've changed it in the last update to use the nil values internally, but don't expose arguments to the function.